src/screens/Episode/helpers/getTime.ts

```diff
@@ -0,0 +1,26 @@
+import * as dateFns from 'date-fns'
+import { DateFormatType, getFormattedDate } from 'src/helpers'
+import { UNIX_TIME_START, TIME_SEPARATOR } from '../common'
+
+const checkHourExist = (hours: string): boolean => {
+  return hours !== '00'
+}
+
+const getTime = (timestamp: number): string => {
```

Comment:
It's not clear what time we will get

```suggestion
const getPlayerTime = (timestamp: number): string => {
```